build_versions.json

```diff
@@ -0,0 +1,8 @@
+{
+  "include": [
+    {
+      "rubygem_r10k": "4.1.0",
```

Comment:
we did the 5.0.0 release yesterday, do you want to update?

Reply:
sounds good, will do